src/cgroup.rs

```diff
@@ -30,16 +28,37 @@ use std::path::{Path, PathBuf};
 /// > specialized behaviour.
 ///
 /// This crate is an attempt at providing a Rust-native way of managing these cgroups.
-pub struct Cgroup<'b> {
+#[derive(Debug)]
+pub struct Cgroup {
     /// The list of subsystems that control this cgroup
     subsystems: Vec<Subsystem>,
 
     /// The hierarchy.
-    hier: Box<&'b dyn Hierarchy>,
+    hier: Box<dyn Hierarchy>,
     path: String,
 }
 
-impl<'b> Cgroup<'b> {
+impl Clone for Cgroup {
```

Comment:
Could it be implemented for Cgroup deriving the Clone/Default?

Reply:
No, because the member hier can not imply Clone trait for it is a trait object.